src/App.js

```diff
@@ -9,10 +9,18 @@ import NotificationsPortal from '@redhat-cloud-services/frontend-components-noti
 import { notificationsReducer } from '@redhat-cloud-services/frontend-components-notifications/redux';
 import { systemRecsReducer, systemDetailReducer } from './store/reducers';
 import { register } from './store';
+import Cookies from 'js-cookie';
+import queryString from 'query-string';
 
 class App extends Component {
 
     componentDidMount () {
+        const params = queryString.parse(this.props.location.search);
```

Comment:
No need to use `queryString` [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) should be sufficient
```suggestion
        const params = new URLSearchParams(this.props.location.search);
```